src/components/Header/Header.tsx

```diff
@@ -0,0 +1,67 @@
+import { useContext, useState } from 'react';
+import { AuthContext } from '../Auth/AuthContext';
+import { postTodo } from '../../api/todos';
+
+import { Error } from '../../types/Error';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  setTodos: (arg: Todo[]) => void,
+  setIsError: (arg: Error | null) => void,
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  setTodos,
+  setIsError,
+}) => {
+  const [query, setQuery] = useState('');
+  const user = useContext(AuthContext);
+
+  const postTodosData = async () => {
+    if (user && query) {
+      const data = {
+        title: query,
+        userId: user.id,
+        completed: false,
+      };
+
+      await postTodo(data)
+        .then(todo => {
+          if (todos) {
+            setTodos([...todos, todo]);
+          } else {
+            setTodos([todo]);
+          }
+        })
+        .catch(() => {
+          setIsError(Error.Add);
+        })
+        .finally(() => {
+          setQuery('');
+        });
+    }
+  };
+
+  return (
+    <header className="header">
+      <h1>{`${user?.name}: todos`}</h1>
```

Comment:
Looks not good to me:
![image](https://user-images.githubusercontent.com/86923089/222956465-58c1ff3d-7ded-4d17-81eb-6821fada6951.png)

Fix styles if you want to show user name here